lib/feedjira/core_ext/time.rb

```diff
@@ -20,7 +20,8 @@ def self.parse_safely(dt)
       parse_string_safely dt.to_s
     end
   rescue StandardError => e
-    Feedjira::Logger.exception(e) { "Failed to parse time #{dt}" }
+    Feedjira.logger.fatal("Failed to parse time #{dt}")
+    Feedjira.logger.fatal(e)
     nil
   end
 
```

Comment:
It's not only a feeling - I've measured that in our app using feedjira. After implementing logging (with a lot of debug outputs) we've found higher CPU Usage than before - and after some investigation, we've replaced all the method calls with blocks - Problem solved.